forge-gui/res/cardsfolder/upcoming/sarinthian_greatwurm.txt

```diff
@@ -0,0 +1,10 @@
+Name:Sarinthian Greatwurm
+ManaCost:4 R G
+Types:Creature Wurm
+PT:7/6
+K:Trample
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Land.YouCtrl | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever a land enters the battlefield, create a tapped Powerstone token. 
```

Comment:
Wrong ValidCard

Reply:
Oh god didn't realize how insane this card is lol